Files/Files.csproj

```diff
@@ -1465,7 +1465,7 @@
       <Version>7.1.1</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.7.0</Version>
+      <Version>2.8.0-prerelease.210927001</Version>
```

Comment:
The update to a prerelease version is intentional, right?

Reply:
Yes